tethys_portal/settings.py

```diff
@@ -12,41 +12,76 @@
 
```

Comment:
I think we ought to add a big warning at the top of this file, reminding people not to edit this file directly and pointing them to the documentation for configuring settings.